USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -0,0 +1,250 @@
+                    }
+                    break;
+                case QMarker qMarker:
+                    XWPFParagraph poetryParagraph = newDoc.CreateParagraph();
+
+                    // Not sure if indentation works
+                    poetryParagraph.IndentationLeft =qMarker.Depth;
+
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker,poetryParagraph);
+                    }
+
+                    break;
+                case MMarker mMarker:
+                    break;
+                case TextBlock textBlock:
+                    XWPFRun blockText = parentParagraph.CreateRun();
+                    blockText.SetText(textBlock.Text);
+                    blockText.FontSize = 16;
+
+                    if (isBold)
+                    {
+                        blockText.IsBold = true;
+                    }
+                    if (isItalics)
+                    {
+                        blockText.IsItalic = true;
+                    }
+
+                    break;
+                case BDMarker bdMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker,parentParagraph,true);
+                    }
+                    break;
+                case HMarker hMarker:
+                    XWPFParagraph newHeader = newDoc.CreateParagraph();
+                    XWPFRun headerTitle = newHeader.CreateRun();
+                    headerTitle.SetText(hMarker.HeaderText);
+                    headerTitle.FontSize = 24;
+                    break;
+                case MTMarker mTMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker);
+                    }
+                    if (!separateChapters)   // No double page breaks before books
+                    {
+                        newDoc.CreateParagraph().CreateRun().AddBreak(BreakType.PAGE);
+                    }
+                    break;
+
+                case FMarker fMarker:
+                    StringBuilder footnote = new StringBuilder();
+
+                    string footnoteId;
+                    switch (fMarker.FootNoteCaller)
+                    {
+                        case "-":
+                            footnoteId = "";
+                            break;
+                        case "+":
+                            footnoteId = $"{FootnoteTextTags.Count + 1}";
+                            break;
+                        default:
+                            footnoteId = fMarker.FootNoteCaller;
+                            break;
+
+                    }
+
+                    XWPFRun footnoteMarker = parentParagraph.CreateRun();
+
+                    footnoteMarker.SetText($"{footnoteId}");
+                    footnoteMarker.Subscript = VerticalAlign.SUBSCRIPT;
```

Comment:
Are you sure you want subscript here and not superscript?

Reply:
I didn't want to confuse the footnote caller with the verse marker, but you're right it should be superscript. -- Changed to superscript